# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/webkitgtk41/no-ruby.diff
# Copyright (C) 2021 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- webkitgtk-2.34.0/Source/cmake/WebKitFeatures.cmake.vanilla 2021-10-01 18:01:01.463496556 +0200
+++ webkitgtk-2.34.0/Source/cmake/WebKitFeatures.cmake 2021-10-01 18:01:28.694497960 +0200
@@ -98,7 +98,7 @@
         message(STATUS "Unified builds are disabled when analyzing sources")
         set(ENABLE_UNIFIED_BUILDS_DEFAULT OFF)
     else ()
-        set(ENABLE_UNIFIED_BUILDS_DEFAULT ON)
+        set(ENABLE_UNIFIED_BUILDS_DEFAULT OFF)
     endif ()

     WEBKIT_OPTION_DEFINE(ENABLE_3D_TRANSFORMS "Toggle 3D transforms support" PRIVATE ON)
